x11-misc/slop/slop-4.3.21.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="An application that queries the user for a selection for printing"
+HOMEPAGE="https://github.com/naelstrof/slop"
+SRC_URI="https://github.com/naelstrof/slop/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="opengl"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXext
+
+	opengl? (
+		media-libs/glew:*
```

Comment:
I have to admit I don't really get the difference of Slots and SubSlots, but maybe glew is a bad example, I wrote me a todo to figure that out someday soon.